### PR TITLE
dotnet-sdk: Update the manifest to fit the dotnet-sdk-sts version

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [x] ♻️ Refactor
- [ ] 🐛 Bug Fix
- [x] 👷 Optimization
- [ ] 🚩 Other

## Description
Change the manifest to be similar of the dotnet-sdk-sts version. Also adding the arm64 support. These change will add some feature to work side-by-side with the dotnet-sdk-sts version.

## Related Tickets & Documents
None

## [optional] What gif/emoji best describes this PR or how it makes you feel?
:technologist: :safety_vest: :wrench: :ambulance:

### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Notes in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Application Dependencies?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Application Suggestions?
- [x] 👍 yes (With a little update)
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Pre-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Post-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### File persistence to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Other ?
- [x] 🚨 Change the Environment variable required (to help working if used side-by-side with dotnet-sdk-sts)
- [x] Added the arm64
- [x] Changes the regex to obtain the version so it wil remove risks of getting any "LTS" version (aka "eol", "maintenance", etc.)

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### dotnet-sdk

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
